### DISCUSSION TITLE
To Zip or Not to Zip the MRFs

### DISCUSSION THREAD

**Discussion by AJLgeHG** [CMSgov/price-transparency-guide#333] [Q&A] [answered]
Hello - When posting links to the MRFs on the Issuer website, are we allowed to post zipped files?

I've seen posts where folks have said they'll zip their files, but I'm not sure that would not be problematic for retrievers of the files. 

Could someone at CMS @shaselton please provide guidance on this question?

Thanks.

**Comment by Devyandu** [comment, marked as answer]
Yes, they can be zipped.
Please see previous reply by Scott.

[https://github.com/CMSgov/price-transparency-guide/discussions/59#discussioncomment-642881](https://github.com/CMSgov/price-transparency-guide/discussions/59#discussioncomment-642881)

**Reply by shaselton-usds** [reply]
Thanks!

**Comment by garysolebc** [comment]
If a file is zipped rather than tarred - should there be something that describes to a client machine the format of the compression?  These files will likely be read by a client that is not only reading our files, but other plan files that may have selected a different compression technology.  Or, will the file extension need to reflect the compression type - and would this mean we need to include it in the file naming convention?

**Reply by shaselton-usds** [reply]
There is no naming standard on the resulting file after compression.

I would imagine the way to go about this in an automated way would be to read the file's extension and respond appropriately.
